src/calendar/header/index.js

```diff
@@ -97,6 +98,29 @@ class CalendarHeader extends Component {
     return this.addMonth();
   }
 
+  renderWeekDays = (weekDaysNames) => {
+    const {disabledDaysIndexes} = this.props;
+    return weekDaysNames.map((day, idx) => {
+      const dayStyle = [this.style.dayHeader];
+      if (disabledDaysIndexes && disabledDaysIndexes.length) {
```

Comment:
you can avoid this check by setting a default value for `disabledDaysIndexes` with empty array
or use `lodash` which knows to handle undefined values